### PR TITLE
handle vmagent tmpDataPath

### PR DESCRIPTION
**Pull request by DisasteR**
manage creation and permission of tmpDataPath

### CODE REVIEW

File: roles/vmagent/defaults/main.yml [hunk 1]
```diff
@@ -27,7 +31,7 @@ vmagent_scrape_config:
 
 vmagent_aggregation_config: []
 
-vmagent_install_download_to_control: true
```
Comment (zekker6) [inline roles/vmagent/defaults/main.yml:30]: Would you mind to move this change to another PR(if it is needed of course)?
It would be great to keep PR implementing/changing only one logical unit at a time to make it easier to track the changes.

**Review by zekker6 — COMMENTED** [review]
@DisasteR Thank you for the PR!
Please, see a comment.

File: roles/vmagent/defaults/main.yml (hunk 1 above)
Comment (DisasteR) [reply roles/vmagent/defaults/main.yml:30]: Hi,

I changed this to match other roles default config for downloading

Regards,

**Review by zekker6 — APPROVED** [review]
